homeassistant/components/seventeentrack/sensor.py

```diff
@@ -126,12 +133,14 @@ def __init__(self, data, status, initial_state):
         self._attr_unique_id = f"summary_{data.account_id}_{slugify(status)}"
 
     @property
-    def available(self):
+    def available(self) -> bool:
         """Return whether the entity is available."""
         return self._state is not None
 
     @property
-    def native_value(self):
+    def native_value(
+        self,
+    ) -> StateType | str | int | float | None | date | datetime | Decimal:
```

Comment:
```suggestion
    ) -> StateType:
```
I think this should be enough